supply-chain-security/compromises/README.md

```diff
@@ -8,16 +8,26 @@ can better understand the patterns and develop best practices and tools.
 
 We welcome additions to this catalog by 
 [filing an issue](https://github.com/cncf/sig-security/issues/new/choose) or
-[github pull request](https://github.com/cncf/sig-security)
+submitting a [pull request](https://github.com/cncf/sig-security) to our repo.
+
+Additions aren't intended to be a statement on the security stance and/or
+practices of any given project, or of open source as a whole. If anything,
+many of the stories showcase rapid response, evolving security practices in
+the community and an open and transparent approach to disclosure.
 
 
 | Name              | Year               | Type of compromise    | Link        |
 | ----------------- | ------------------ | ------------------    | ----------- |
 | [electron-native-notify](2019/electron-native-notify.md) | 2019 | Source Code Compromise | [1](https://blog.npmjs.org/post/185397814280/plot-to-steal-cryptocurrency-foiled-by-the-npm), [2](https://komodoplatform.com/update-agama-vulnerability/)|
+| [PyPI typosquatting](2019/pypi.md) | 2019 | Naming attack | [1](https://blog.reversinglabs.com/blog/suppy-chain-malware-detecting-malware-in-package-manager-repositories) |
+| [ROS build farm compromise](2019/ros.md) | 2019 | Build infrastructure | [1](https://discourse.ros.org/t/security-issue-on-ros-build-farm/9342/8), [2](https://discourse.ros.org/t/new-gpg-keys-deployed-for-packages-ros-org/9454) |
```

Comment:
change `(Build infrastructure)` to `(Trust and Signing)` and add `(Publishing Infrastructure)` as a new line in the table for consistency with definitions.